labs/cicd-artifact-store/cloudbuild-release.yaml

```diff
@@ -0,0 +1,68 @@
+#  Copyright 2020 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+steps:
+  - id: Copying Apigee artifact
+    name: gcr.io/google.com/cloudsdktool/cloud-sdk
+    entrypoint: bash
+    args:
+      - -c
+      - |
+        gsutil rsync gs://${_APIGEE_BUILD_BUCKET}/${_API_PROJECT}/${_COMMIT_SHA}/ ./
+        mkdir target
+        mv *.zip target
+  - id: Fetch token for deployment
+    name: gcr.io/cloud-builders/gcloud
+    entrypoint: bash
+    args:
+      - -c
+      - |
+        gcloud auth print-access-token > /workspace/.build_token
+  - id: Package deploy and upload artifact
+    name: gcr.io/cloud-builders/mvn
+    entrypoint: bash
+    args:
+      - -c
+      - |
+        mvn install -f pom-release.xml \
+                    -Pprod \
+                    -Dapigee.config.options=update \
+                    -Dorg=${_APIGEE_ORG} -Denv=${_APIGEE_ENV} \
+                    -Dbearer=$(cat /workspace/.build_token)
+  - id: Fetch API key for test
```

Comment:
There's a better integrated and more secure way of doing this now in cloud run: https://cloud.google.com/build/docs/securing-builds/use-secrets

Reply:
This is an awesome update. Can I do this as a separate change, I am sure I will have another round updates as the doc is reviewed.